views/lib/loggedin.hbs

```diff
@@ -110,9 +110,9 @@
                             {{#ifneq (getConfig "GLOBAL_ANNOUNCEMENT_ROLES") ""}}
                                 {{#if (userHasRoleFromArray (getConfig "GLOBAL_ANNOUNCEMENT_ROLES") ../currentUser)}}
                                     <section class="demo">
-                                        <div class="alert alert-info" role="alert">
-                                            <div>{{{getConfig "GLOBAL_ANNOUNCEMENT_TEXT"}}}</div>
-                                        </div>
+                                            <div class="alert alert-info" role="alert" style="background-color: #D9EDF7">
```

Comment:
I know its the legacy client but it looks like a magic color.. can you put it in the style.scss ?

Reply:
see rocketchat, confirmation from UX needed in my opinion

Reply:
I have created alert-annoncement class